apis/eventstream/index.yaml

```diff
@@ -89,7 +89,7 @@ get:
             light_client_optimistic_update:
               description: The node's latest known `LightClientOptimisticUpdate` has been updated
               value: |
-                event: light_client_optimimstic_update
+                event: light_client_optimistic_update
```

Comment:
Is renaming the event a potential problem? I checked the [lodestar](https://github.com/ChainSafe/lodestar) implementation and they use the proper value `light_client_optimistic_update` without the typo in the code.

Reply:
IMO renaming to remove a typo is OK, if a client actually implemented the typo'd version I think it would be sensible to deprecate/remove the typo version.

Reply:
Ok with removing the typo :+1: